README.md

```diff
@@ -1,66 +1,128 @@
-(Note: the *Allowed Origins* and *Allowed Return URLs* should use **http**, not https.)  
+   for the xCORE VocalFusion XVF3510,
 
-5. Run the installation script: `source ~/vocalfusion-avs-setup/auto_install.sh`    
-Read and accept the AVS Device SDK license agreement.
+   ```./auto_install.sh xvf3500```
 
-6. You will be prompted enter your Alexa device details and asked whether you want the Sample App to run automatically when the Raspberry Pi boots.
+   for the xCORE VocalFusion XVF3500, or
 
-7. Read and accept the Sensory license agreement.
-Wait for the script to complete the installation. This can take some time.
+   ```./auto_install.sh xvf3100```
 
-8. As a final step, the script will open http://localhost:3000 in a browser on the Raspberry Pi. Enter your Amazon Developer credentials and close the browser window when prompted. (You won't have to do this if you already have a valid configuration file.)  
-If you see a `400 Bad Request - HTTP` error it may be necessary to add http:// URLs to your device origins and return fields. To do this, go to your amazon developer AVS home page (https://developer.amazon.com/avs/home.html) and from `My products` select `manage` and then `security profile`. Now add http://localhost:3000 to the `Allowed origins` and http://localhost:3000/authresponse to the `Allowed return URLs`. Now refresh the browser window with the original error.
+   for the xCORE VocalFusion XVF3100.
 
-9. Enter `sudo reboot` to reboot the Raspberry Pi and complete the installation.
+   Read and accept the AVS Device SDK license agreement.
+
+8. You will be asked whether you want the Sample App to run automatically when the Raspberry Pi boots. It is recommended that you respond "yes" to this option.
+
+9. Read and accept the Sensory license agreement. Wait for the script to complete the installation. The script is configuring the Raspberry Pi audio system, downloading and updating dependencies, building and configuring the AVS Device SDK. It takes around 30 minutes to complete.
+
+10. Enter `sudo reboot` to reboot the Raspberry Pi and complete the installation.
+
+11. If you selected the option to run the Sample App on boot you should now be able to complete the registration by following the steps from 2 onward here:
+https://github.com/alexa/avs-device-sdk/wiki/Raspberry-Pi-Quick-Start-Guide-with-Script#finish-authorization-using-login-with-amazon
+
+12. Now you can execute an AVS command such as "Alexa, what time is it?".
+
+   On the XMOS **xCORE VocalFusion XVF3510 Kit for Amazon AVS**, the LED on the Pi HAT board will change colour when the system hears the "Alexa" keyword, and will then cycle back and forth whilst waiting for a response from the Amazon AVS server.
+
+   On the XMOS **xCORE VocalFusion Stereo 4-Mic Kit for Amazon AVS**, the LEDs on the development board should reflect the approximate direction from which the microphones are receiving a stimulus.
```

Comment:
can these line be valid for the mono kit as well?

Reply:
I've added in a reference to the mono kit..